src/ui/public/agg_types/controls/top.html

```diff
@@ -0,0 +1,29 @@
+<div class="form-group">
+  <label for="sort">
+    Sort On
+  </label>
+
+  <select
+    class="form-control"
+    name="sortField"
+    required
+    ng-model="agg.params.sortField"
+    ng-show="indexedFields.length"
+    auto-select-if-only-one="indexedFields"
+    ng-options="field as field.displayName group by field.type for field in indexedFields">
+  </select>
```

Comment:
It is not possible to have the metric use two `field` templates because the same model [`agg.params.field`](https://github.com/Bargs/kibana/blob/70ba64fba2672e4c2937aa2ad0f588bc1f105263/src/ui/public/agg_types/controls/field.html#L24) is used.
Also, I wanted to have a custom label, i.e., `Sort on`.

Maybe I can update that template so that:

1. the label can be set optionally to some custom text
1. if the parameter's name is set to something other than `field`, e.g., [`sortField`](https://github.com/elastic/kibana/pull/7302/files#diff-76a0c58b4fbf5f7e9274f385ed545048R26), then the model object used would be `agg.params.sortField` instead of `agg.params.field`.